src/roadrunner-symfony-nyholm/src/Runner.php

```diff
@@ -30,13 +32,21 @@ class Runner implements RunnerInterface
      */
     private $sessionOptions;
 
-    public function __construct(KernelInterface $kernel, ?HttpFoundationFactoryInterface $httpFoundationFactory = null, ?HttpMessageFactoryInterface $httpMessageFactory = null)
+    public function __construct(HttpKernelInterface $kernel, ?HttpFoundationFactoryInterface $httpFoundationFactory = null, ?HttpMessageFactoryInterface $httpMessageFactory = null)
```

Comment:
Hm.. Lets drop this type hint and add `HttpKernelInterface|KernelInterface` in the doc block

Reply:
Thx for fixing the typehint!